ansible/group_vars/all/defaults.yaml

```diff
@@ -58,7 +58,7 @@ aws_images: []
 
 extra_images: []
 
-containerd_supplementary_images: ["docker.io/mesosphere/toml-merge:v0.1.0"]
+containerd_supplementary_images: ["ghcr.io/mesosphere/toml-merge:v0.2.0", "ghcr.io/mesosphere/dynamic-credential-provider:v0.2.0"]
```

Comment:
This looks good for building AMI and other cloud images when not air-gapped.
We do need to to also update the toml-merge image version and add [the new image in here](https://github.com/mesosphere/dkp-release/blob/a72a037fab7fd7bb9772bc352f5191e2539123fb/k8s-images-bundles/Makefile#L44).

Let's wait to merge this please until we have a new the RC cut.

Reply:
I think we can merge this once we have a `release-2.2` branch.

Reply:
Images were updated and merged in mesosphere/dkp-release#66